apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.tsx

```diff
@@ -177,7 +177,7 @@ const OperationD = (props: Props) => {
   const urlFeesInfo = getURLFeesInfo
     ? getURLFeesInfo({ op: operation, currencyId: cryptoCurrency.id })
     : null;
-  const url = getTransactionExplorer(getDefaultExplorerView(mainAccount.currency), operation.hash);
+  const url = cryptoCurrency.id != "hedera" ? getTransactionExplorer(getDefaultExplorerView(mainAccount.currency), operation.hash) : getTransactionExplorer(getDefaultExplorerView(mainAccount.currency), operation.extra.consensusTimeStamp);
```

Comment:
Unfortunately using a coin-specific condition here is not allowed, you should use our component customization mechanism for this. You can look up for the `LLDCoinFamily` type for examples.

Reply:
@Sheng-Long This type was introduced recently, it's on develop [here](https://github.com/LedgerHQ/ledger-live/blob/4ec6a773817d070c6cc7b0db3c27ab54328e8ac0/apps/ledger-live-desktop/src/renderer/families/types.ts#L75).
If you update our branch on develop you should find it, along with how it's used in other families.

Reply:
Hi @Sheng-Long we have added some documentation regarding the component customization: https://developers.ledger.com/docs/coin/ui-ux-guidelines/

Please let me know if you need more help on this.